pkgs/build-support/rust/sysroot/update-lockfile.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p python3 python3.pkgs.toml cargo
+
+set -eu pipefile
+
+HERE=$(readlink -e $(dirname "${BASH_SOURCE[0]}"))
+NIXPKGS_ROOT="$HERE/../../../.."
+
+# https://unix.stackexchange.com/a/84980/390173
+tempdir=$(mktemp -d 2>/dev/null || mktemp -d -t 'update-lockfile')
```

Comment:
Is this not using nix' coreutils? Maybe we need to add them to the packages for the shell?

Reply:
N.B. this was in the script before I moved it so I soon as well deal with later.